src/types/account-manager.ts

```diff
@@ -257,22 +388,33 @@ export class AccountManager {
    * @returns The account
    */
   public random() {
-    return this.signerAccount(randomAccount())
+    return this.signerAccount(algosdk.generateAccount())
   }
 
   /**
-   * Returns an account (with private key loaded) that can act as a dispenser.
+   * Returns an account (with private key loaded) that can act as a dispenser from
+   * environment variables, or against default LocalNet if no environment variables present.
+   *
+   * Note: requires a Node.js environment to execute.
+   *
+   * It present, it will load the account mnemonic stored in process.env.DISPENSER_MNEMONIC and optionally
+   * process.env.DISPENSER_SENDER if it's a rekeyed account.
    *
    * @example
    * ```typescript
-   * const account = await account.dispenser()
+   * const account = await account.dispenserFromEnvironment()
    * ```
-   * If running on LocalNet then it will return the default dispenser account automatically,
-   *  otherwise it will load the account mnemonic stored in process.env.DISPENSER_MNEMONIC.
+   *
    * @returns The account
    */
-  public async dispenser() {
```

Comment:
This is a breaking naming change. Previously it was called `dispenser`, but now it's called `dispenserFromEnvironment`, which matches the convention that anything that pulls from environment variables now has "FromEnvironment" in the name. These methods all need Node.js, so having a clear demarcation of this is important I think.

Given this class is part of the new AlgorandClient ecosystem that is marked as beta, it's expected there may be some breaking changes so I think it's OK to do this without bumping major version, but happy if anyone disagrees with this.

Reply:
Fine by me.